integrations/braze/build.gradle.kts

```
@@ -0,0 +1,74 @@
+import io.gitlab.arturbosch.detekt.Detekt
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.detekt)
+    alias(libs.plugins.kotlin.serialization)
+}
+
+detekt {
+    config.setFrom("$rootDir/config/detekt/detekt.yml")
+    buildUponDefaultConfig = true
+    autoCorrect = true
+    parallel = true
+}
+
+tasks.withType<Detekt>().configureEach {
+    reports {
+        html.required.set(true) // observe findings in your browser with structure and code snippets
+    }
+}
+
```

```suggestion
tasks.withType<Test> {
    useJUnitPlatform()
    testLogging {
        events("failed")
    }
}
```
Add this to make the test runnable. Currently, tests are not running.